gocloak.go

```diff
@@ -128,6 +128,12 @@ type GoCloak interface {
 	// AddRealmRoleComposite adds roles as composite
 	DeleteRealmRoleComposite(token string, realm string, roleName string, roles []Role) error
 
+	// *** Client Roles ***
```

Comment:
Please add this functions to `README.md`

Reply:
I just updated README.md, I'll try to write test, I don't know if I'll have time today.